docs/src/howto-guides/016_apama_software_management_plugin.md

```diff
@@ -0,0 +1,89 @@
+# Apama Software Management Plugin
+
```

Comment:
isn't it better to mention that apama should be running on the system to use this plugin.

Reply:
Apama need not be running. It just needs to be installed and it'll be started on the installation of a project and it'll be stopped when the project is removed.